src/exchange/liquidation_handler.cairo

```diff
@@ -110,15 +120,40 @@ mod LiquidationHandler {
     //                          EXTERNAL FUNCTIONS
     // *************************************************************************
     #[external(v0)]
-    impl LiquidationHandlerImpl of super::ILiquidationHandler<ContractState> {
+    impl LiquidationHandlerImpl of super::ILiquidationHandler<ContractState> { // executes a position liquidation
         fn execute_liquidation(
             ref self: ContractState,
             account: ContractAddress,
             market: ContractAddress,
             collateral_token: ContractAddress,
             is_long: bool,
             oracle_params: SetPricesParams
-        ) { // TODO
+        ) {
+            let starting_gas: u128 = starknet_utils::sn_gasleft(array![100]);
+            let mut state_base: BaseOrderHandler::ContractState =
+                BaseOrderHandler::unsafe_new_contract_state(); //retrieve BaseOrderHandler state
+            let key: felt252 = create_liquidation_order(
+                state_base.data_store.read(),
+                state_base.event_emitter.read(),
+                account,
+                market,
+                collateral_token,
+                is_long
+            );
+            let tmp_oracle_params: SetPricesParams = oracle_params.clone();
+            let params: ExecuteOrderParams =
+                BaseOrderHandler::InternalImpl::get_execute_order_params(
+                ref state_base,
+                key,
+                tmp_oracle_params,
+                account,
+                starting_gas,
+                SecondaryOrderType::None
+            );
+            validate_feature(state_base.data_store.read(), key);
+            let mut state_order: OrderHandler::ContractState =
```

Comment:
Use directly `order_utils::execute_order(params)` instead of using an order handler